src/store/blockstore-level.ts

```diff
@@ -52,6 +52,8 @@ export class BlockstoreLevel implements Blockstore {
    * releases all file handles and locks held by the underlying db.
    */
   async close(): Promise<void> {
+    await this.createLevelDatabase();
```

Comment:
for my understanding: why is it not easier/better to just not do anything if `db` is not defined?

Reply:
oh lol true didnt think of that